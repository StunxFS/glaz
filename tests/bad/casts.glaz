fn main() {
  unsafe { _ = cast(3, rawptr); }
}
