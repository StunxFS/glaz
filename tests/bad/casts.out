tests/bad/casts.glaz:2:17: error: cannot cast type `untyped_int` to `rawptr`
     | 
   2 |   unsafe { _ = cast(3, rawptr); }
     |                ^^^^^^^^^^^^^^^
glaz error: could not compile `casts`, aborting due to previous error.
